### PR TITLE
Feature/frontend/web/angular matthew schoeman

### PR DESCRIPTION
**Pull request by MTSchoeman**

### CODE REVIEW

File: Web/Angular/GOT/src/app/components/base-book/base-book.component.css
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/base-book/base-book.component.css:1]: Please may you remove the empty files

File: Web/Angular/GOT/src/app/components/base-house/base-house.component.css
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/base-house/base-house.component.css:1]: Please remove this empty file

File: Web/Angular/GOT/src/app/components/book-details/book-details.component.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.card {
+  border: 0px solid black;
```
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/book-details/book-details.component.css]: Does this 0px render anything?

File: Web/Angular/GOT/src/app/components/book/book.component.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+              <div class="card-body">
+                  <div class="row">
+                      <div class="col-md-6">
+
+                          <h4>Title: </h4>
+                          <h6>{{book.name ? book.name : 'Unknown'}}</h6>
+                      </div>
+                      <div class="col-md-6">
+
+                          <h4>ISBN</h4>
```
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/book/book.component.html]: For consistency is this meant to have  a colon?

File: Web/Angular/GOT/src/app/components/book/book.component.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+              </div>
+          </div>
+      </div>
+  </div>
+  <ng-template #isLoading>
+      <div class="text-center">
+          <h3 [ngStyle]="{'color' : 'white'}">Busy Fetching The Might Houses</h3>
+      </div>
+  </ng-template>
+  <br><br><br><br><br><br><br><br>
```
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/book/book.component.html]: Is this here to add spacing? If so please use css to generate spacing

File: Web/Angular/GOT/src/app/components/book-details/book-details.component.css (hunk 1 above)
Comment (MTSchoeman) [reply Web/Angular/GOT/src/app/components/book-details/book-details.component.css]: By default, the card component in Bootstrap has a border, so I used 0px to hide it

File: Web/Angular/GOT/src/app/components/character/character.component.css
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/character/character.component.css:1]: Please delete this empty file

File: Web/Angular/GOT/src/app/components/characters/characters.component.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                        </div>
+                        <div class="col-md-6">
+
+                            <h4>Culture: </h4>
+                            <h6>{{character.culture ? character.culture : 'Culture not specified'}}</h6>
+                        </div>
+                        <hr>
+                        <div class="col-xs-12">
+
+                            <h4>Born:</h4>
```
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/characters/characters.component.html]: Please be consistent with the spacing after the colon

File: Web/Angular/GOT/src/app/components/house-details/house-details.component.css [hunk 5]
```diff
@@ -0,0 +1,32 @@
+.card {
+  border: 0px solid black;
```
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/house-details/house-details.component.css]: Same question regarding the 0px border width. If the border is not meant to rendered can you go border-style: none

File: Web/Angular/GOT/src/app/components/house-details/house-details.component.html [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+                  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-4 my-2"><label>Founded: </label> <br> {{selectedHouse.founded ?
+                    selectedHouse.founded :
+                    'Unknown'}}</div>
+                    <div class="col-xs-12 col-sm-6 col-md-6 col-lg-4 my-2"><label>Died Out: </label> <br> {{selectedHouse.diedOut ?
+                      selectedHouse.diedOut :
+                      'Unknown'}}</div>
+                      <div class="col-xs-12 col-sm-6 col-md-6 col-lg-4 my-2"><label>House Words: </label> <br> {{selectedHouse.words ?
+                        selectedHouse.words :
+                        'Unknown'}}</div>
+                      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 my-2"><label>Coat of Arms:</label> <br> {{selectedHouse.coatOfArms ?
```
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/house-details/house-details.component.html]: Please be consistent with spacing after the colon

File: Web/Angular/GOT/src/app/components/house/house.component.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+                        </div>
+                        <div class="col-md-6">
+
+                            <h4>Words: </h4>
+                            <h6>{{house.words ? house.words : 'Unknown'}}</h6>
+                        </div>
+                        <hr>
+                        <div class="col-xs-12">
+
+                            <h4>Coat of Arms:</h4>
```
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/house/house.component.html]: Ditto

File: Web/Angular/GOT/src/app/components/pagination/pagination.component.css
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/pagination/pagination.component.css:1]: Please delete empty file

File: Web/Angular/GOT/src/app/components/house/house.component.css [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+#sticky-container {
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 9999;
+  padding: 1rem;
+  background-color: #21252957;
+}
+/*
```
Comment (KyleO249) [inline Web/Angular/GOT/src/app/components/house/house.component.css]: Ghost code

File: Web/Angular/GOT/src/app/components/base-book/base-book.component.css
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/base-book/base-book.component.css:1]: Resolved

File: Web/Angular/GOT/src/app/components/base-house/base-house.component.css
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/base-house/base-house.component.css:1]: Resolved

File: Web/Angular/GOT/src/app/components/book-details/book-details.component.css (hunk 1 above)
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/book-details/book-details.component.css]: Resolved

File: Web/Angular/GOT/src/app/components/book/book.component.html (hunk 2 above)
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/book/book.component.html]: Resolved

File: Web/Angular/GOT/src/app/components/book/book.component.html (hunk 3 above)
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/book/book.component.html]: Resolved

File: Web/Angular/GOT/src/app/components/character/character.component.css
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/character/character.component.css:1]: Resolved

File: Web/Angular/GOT/src/app/components/characters/characters.component.html (hunk 4 above)
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/characters/characters.component.html]: Resolved

File: Web/Angular/GOT/src/app/components/house-details/house-details.component.css (hunk 5 above)
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/house-details/house-details.component.css]: Resolved

File: Web/Angular/GOT/src/app/components/house-details/house-details.component.html (hunk 6 above)
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/house-details/house-details.component.html]: Resolved

File: Web/Angular/GOT/src/app/components/house/house.component.html (hunk 7 above)
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/house/house.component.html]: Resolved

File: Web/Angular/GOT/src/app/components/pagination/pagination.component.css
Comment (KyleO249) [reply Web/Angular/GOT/src/app/components/pagination/pagination.component.css:1]: Resolved

**Review by KyleO249 — APPROVED** [review]

**Pull request closed without merging**